recipes/pyface/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set version = "5.0.0" %}
+
+package:
+  name: pyface
+  version: {{ version }}
+
+source:
+  fn: pyface-{{ version }}.tar.bz2
+  url: https://pypi.python.org/packages/source/p/pyface/pyface-{{ version }}.tar.bz2
+  md5: 0468d6a44adba0b70ec5171d1af9350a
+  patches:
+    - pyface_fix.patch
```

Comment:
Can you please reference where this patch originated in a comment? Probably mention the issue (with ref) and the commit (also with ref) you used.